code/modules/mining/lavaland/world_anvil.dm

```diff
@@ -31,76 +45,29 @@
 		to_chat(user,"You place down the gibtonite on the world anvil, and watch as the gibtonite melts into it. The world anvil is now heated enough for [forge_charges] forge[forge_charges > 1 ? "s" : ""].")
```

Comment:
If we're going to capitalize instances of "World Anvil", we'll need to be comprehensive. Check upwards through the other references in this file too.

Reply:
```suggestion
		to_chat(user,"You place down the gibtonite on the World Anvil, and watch as the gibtonite melts into it. The World Anvil is now heated enough for [forge_charges] forge[forge_charges > 1 ? "s" : ""].")
```